x-pack/test/functional/apps/dashboard_mode/dashboard_empty_screen.js

```diff
@@ -36,22 +36,19 @@ export default function ({ getPageObjects, getService }) {
       await PageObjects.visualize.clickLensWidget();
       await PageObjects.lens.goToTimeRange();
       await PageObjects.lens.configureDimension({
-        dimension:
-          '[data-test-subj="lnsXY_xDimensionPanel"] [data-test-subj="lns-empty-dimension"]',
+        dimension: 'lnsXY_xDimensionPanel > lns-empty-dimension',
```

Comment:
I recently read through the source code for our `testSubjects` implementation, and found out that these are equivalent.